six/common/builtins/datadog_agent.c

```diff
@@ -334,7 +334,7 @@ static PyObject *set_external_tags(PyObject *self, PyObject *args)
         for (j = 0; j < tags_len; j++) {
             PyObject *s = PyList_GetItem(value, j);
             if (s == NULL) {
-                continue;
+                goto error;
```

Comment:
This is on purpose: we ignore invalid tags. This was the previous behavior

Reply:
Correct me if I am wrong but an invalid tag wouldn't result in a `PyList_GetItem` failure right ? Instead we would fail to convert it to string here:
https://github.com/DataDog/datadog-agent/blob/b9c075ecc85ee3fa668057156fe7559da5981d19/six/common/builtins/datadog_agent.c#L341-L344

Reply:
Indeed, but we might want to send the tags we already collected. So either a break or a continue.

Reply:
I think a `break` here makes sense.